HitScoreVisualizer/Services/ConfigProvider.cs

```diff
@@ -229,6 +229,17 @@ private static bool Validate(Configuration configuration, string configName)
 				return false;
 			}
 
+			// 99 is the max for NumberFormatInfo.NumberDecimalDigits
+			if (configuration.TimeDependenceDecimalPrecision < 0 || configuration.TimeDependenceDecimalPrecision > 99)
+			{
+				return false;
```

Comment:
We're getting really close now, but please also log why it failed the validation so a user can look up in the logs to see what they would have to fix 🙂